code/datums/storage/storage.dm

```diff
@@ -132,15 +132,18 @@
 	orient_to_hud()
 
 /datum/storage/Destroy()
+	for(var/mob/person in is_using)
+		if(person.active_storage == src)
+			person.active_storage = null
+			if(person.client)
+				person.client.screen -= boxes
+				person.client.screen -= closer
```

Comment:
I am FAIRLY CERTAIN this can cause a failed Destroy due to client disappearing mid execution